packages/2d/src/components/Line.ts

```diff
@@ -230,6 +228,16 @@ export class Line extends Shape {
     return box;
   }
 
+  protected validateProps(props: LineProps) {
```

Comment:
Extracted to a method so that `Spline` can override this as both the condition as well as the error message changes.